### PR TITLE
quiz2

### PR DESCRIPTION
**Pull request by jmhjh2015**
sorry about the late.it took me half hour to figure it out how github works

### CODE REVIEW

**Comment by jmhjh2015** [comment]
It was just 5mins late.i guess that is ok
JunHao Huang

**Comment by matty-digital** [comment]
looks like you messed up when pushing to the repo, it's showing additions and deletions

File: quiz2.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ +		color: white;
+ +	font-weight: bold;
+  }
+  
+  .timeout {
+ -	background: #CCCCCC;
+ +	background: #004d26;
+ +		color: white;
+ +	font-weight: bold;
+  }
```
Comment (matty-digital) [inline quiz2.css]: errors in this file look to be mostly fixed, but this file won't run properly due to the extra characters

File: quiz2.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
+<!doctype html>
+ <html lang="en">
+ 	<head>
+ 		<meta charset="UTF-8"
+ 		<title>Quiz 2</title>
+<head>
+ 		<meta charset="UTF-8"
+ 		<meta charset="UTF-8">
```
Comment (matty-digital) [inline quiz2.html]: this meta tag is included 3 times

File: quiz2.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+ <html lang="en">
+ 	<head>
+ 		<meta charset="UTF-8"
+ 		<title>Quiz 2</title>
+<head>
+ 		<meta charset="UTF-8"
+ 		<meta charset="UTF-8">
+  		<title>Quiz 2</title>
+ 		<link rel="stylesheet" type="text/css" href="quiz2.css">
+ 		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
```
Comment (matty-digital) [inline quiz2.html]: we want to keep scripts as close to the closing body tag whenever possible

File: quiz2.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+ 		<script src="quiz2.js"></script>
+ 		
+ 		<script>
```
Comment (matty-digital) [inline quiz2.html]: all of this js functionality should have been placed in the js file included

File: quiz2.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+ 				$("#word_display").html(list[Math.floor((Math.random() * list.length) + 1)]);
+ 			}
+ 			
+ 			function feedMeCookies()
+ 			{
+ 				document.cookie="lastEntry="+$("#word_display").html();
+ 			}
+ 
+
+function getTitle()
```
Comment (matty-digital) [inline quiz2.html]: spacing looks way off

File: quiz2.html [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+ 		<div id="content" class="content">	
+ 		
+ 			<button id="but0" onclick="getTitle()">Get Title</button>
+  		
+
+ 		
+ 		<div class="content">
+ 			<h1>Hi, Welcome to the second quiz!</h1>
+                    <h1>Hey,matt.i was in class today.(JunHao Huang)</h1>
```
Comment (matty-digital) [inline quiz2.html]: :D

File: quiz2.html [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+ +						<input value="SUBMIT" type="submit"></input>
+ 					</form>
+ 				</div>
+ 				<div class="timeout" style="display:none">
+ 					<span>timeout function</span>
+ 				</div>
+ 			</div>
+ 		
+ 
+ 		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
```
Comment (matty-digital) [inline quiz2.html]: these scripts are included twice

File: quiz2.js [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+ 	// Whether or not you use jQuery, you still have to fix the jQuery errors. Removing jQuery is not fixing the problem.
+ 
+ 	$mouseover = $('.mouse-over');
+ 	$click     = $('.click');
+ 	//$sub       = $('.submit');
+ 	$timeout = $('.timeout');
+ 
+ 	$mouseover.on('mouseover', function() {
+ 		$this = $(this);
+ 		$(this).html('Me?');
```
Comment (matty-digital) [inline quiz2.js]: lol

**Pull request closed without merging**